bin/dart-petstore.sh

```diff
@@ -28,4 +28,7 @@ fi
 export JAVA_OPTS="${JAVA_OPTS} -XX:MaxPermSize=256M -Xmx1024M -DloggerPath=conf/log4j.properties"
 ags="$@ generate -i modules/swagger-codegen/src/test/resources/2_0/petstore.yaml -l dart -o samples/client/petstore/dart"
 
+# for dart vm lib generation:
+#ags="$@ generate -i modules/swagger-codegen/src/test/resources/2_0/petstore.yaml -l dart -o samples/client/petstore/dart --additional-properties browserClient=false"
+
```

Comment:
And you may consider later adding a CLI option (e.g. dartBrowserClient) instead of using additional properties.